### ISSUE TITLE
Latest release returns error.

### ISSUE THREAD

**Issue by alexandershelega** [dtzar/helm-kubectl#136]
Hello latest docker image has some issues

```
Error: UPGRADE FAILED: error validating "": error validating data: the server responded with the status code 426 but did not return more information```


3.11 works fine.

**Comment by gaby** [comment]
@alexandershelega This version uses k8s 1.27.1, which k8s are you using?

**Comment by gaby** [comment]
@alexandershelega Related issue here: https://github.com/helm/helm/issues/12071 seems to be a bug with Helm

**Comment by alexandershelega** [comment]
@gaby i have a different versions of k8s running, 1.23, 1.25, 1.26 for all versions falls.

**Comment by ceefour** [comment]
Happened to me too:

```
Error: UPGRADE FAILED: error validating "": error validating data: the server responded with the status code 426 but did not return more information
```

Downgrading to Helm 3.11 "fixed" the issue.

**Comment by gaby** [comment]
This will be fixed with 3.12.1 https://github.com/helm/helm/pull/12096

**Comment by Aaron-ML** [comment]
> This will be fixed with 3.12.1 [helm/helm#12096](https://github.com/helm/helm/pull/12096)

Any idea when 3.12.1 is coming out?
